data/json/items/comestibles/med.json

```diff
@@ -1677,6 +1677,24 @@
     "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
     "charges": 10,
     "stack_size": 10
+  },
+    {
+    "id": "sugar_pills",
```

Comment:
```suggestion
    "id": "homeopatic_pills",
```
It's a detail but why not just give it the same id as the name for clarity?

Reply:
Changed names to `homeopathic_pills`.